node/services/userService.js

```diff
@@ -90,6 +90,38 @@ module.exports = {
             });
     },
 
+    createRole: (req, res, next) => {
```

Comment:
I almost feel like `createRole` suggests this endpoint does something it's not. It sounds like we're going to create a role then able to be assigned to possibly one or more people. `createAndAddRoleToUser` seems more appropriate or `postUserRole` or something..

Reply:
Maybe postUserRole ? createAndAddRoleToUser seems a bit verbose, though I do like the clarity.